.copyrightignore

```diff
@@ -4,6 +4,7 @@ _generated.go
 hack/namespace.yaml
 hack/registeredcluster.yaml
 hack/hubconfig.yaml
+hack/clusterregistrar.yaml
 hack/compute/apibinding.yaml
 hack/compute/apiexport.yaml
 hack/compute/role.yaml
```

Comment:
I think at the end, @robinbobbitt would like to have the hack/compute in the repo as examples.

Reply:
added copyright to yamls in hack dir and removed them from .copyrightignore